appveyor.yml

```diff
@@ -73,3 +73,4 @@ build_script:
   - ps: echo "Size of caches (bytes):"
   - ps: Get-ChildItem -Recurse 'C:\maven\apache-maven-3.8.6' | Measure-Object -Property Length -Sum
   - ps: Get-ChildItem -Recurse 'C:\Users\appveyor\.m2' | Measure-Object -Property Length -Sum
+  - ps: CMD: "./.ci/validation.cmd git_diff"
```

Comment:
Please move it up to be above `ps: echo "Size of caches`

Reply:
done